src/screens/SurveyDetail/detail.js

```diff
@@ -44,7 +44,12 @@ const Detail = () => {
         <p className="survey-detail__subtitle">
           { detailsContext.attributes.description }
         </p>
-        <button className="button button--primary survey-detail__action" onClick={ startSurvey }>Start Survey</button>
+        <button
```

Comment:
@rafayet-monon Should we rollback this file?
And once in another PR we have some changes in this file then we can update it ✌️
This updated file seems not relate with the current task 😅